src/vuestic-theme/vuestic-components/vuestic-rating/VuesticRating.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div class="vuestic-rating">
+    <div
+      v-if="numbers === true"
```

Comment:
v-if and v-for directives shouldn't be on one element, see:
https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential